website/pages/en/developing/substreams-powered-subgraphs-faq.mdx

```diff
@@ -0,0 +1,89 @@
+---
+title: Substreams-powered subgraphs FAQ
+---
+
+## What are Substreams?
+
+Developed by [StreamingFast](https://www.streamingfast.io/), Substreams is an exceptionally powerful processing engine capable of consuming rich streams of blockchain data. Substreams allow you to refine and shape blockchain data for fast and seamless digestion by end-user applications. More specifically, Substreams is a blockchain-agnostic, parallelized, and streaming-first engine, serving as a blockchain data transformation layer. Powered by the [Firehose](https://firehose.streamingfast.io), it ​​enables developers to write Rust modules, build upon community modules, provide extremely high-performance indexing, and [sink](/substreams/developers-guide/sink-targets/README/#substreams-sinks-overview) their data anywhere.
+
+Go to the [Substreams Documentation](/substreams/README/) to learn more about Substreams.
+
+## What are Substreams-powered subgraphs?
+
+[Substreams-powered subgraphs](/cookbook/substreams-powered-subgraphs/) combine the power of Substreams with the queryability of subgraphs. When publishing a Substreams-powered Subgraph, the data produced by the Substreams transformations, [formatted in a special way](https://github.com/streamingfast/substreams-sink-entity-changes/blob/develop/substreams-entity-change/src/tables.rs), are inserted into the database as if it was produced by the subgraph itself.
+
+If you are already familiar with subgraph development, then note that Substreams-powered subgraphs can then be queried, just as if it had been produced by the AssemblyScript transformation layer, with all the Subgraph benefits, like time-travel queries.
```

Comment:
I don't think we necessarily want to call out time-travel queries as a primary subgraph benefit (I would emphasize the dynamic and flexible GraphQL API)

Reply:
Good call - fixed that